frontend/src/scenes/surveys/SurveyResponseLimitWidget.tsx

```diff
@@ -3,62 +3,61 @@ import { LemonBanner, Link, Tooltip } from '@posthog/lemon-ui'
 import { useValues } from 'kea'
 import { billingLogic } from 'scenes/billing/billingLogic'
 import { userLogic } from 'scenes/userLogic'
-
-const DEFAULT_SURVEY_RESPONSE_LIMIT = 250
+import { organizationLogic } from 'scenes/organizationLogic'
 
 export function SurveyResponseLimitWidget(): JSX.Element | null {
     const { billing } = useValues(billingLogic)
     const { user } = useValues(userLogic)
+    const { isAdminOrOwner } = useValues(organizationLogic)
 
-    // Only show for non-admin users
-    if (user?.is_staff || user?.is_impersonated) {
+    // Only hide when impersonating, show for all other users (including admins)
+    if (user?.is_impersonated) {
         return null
     }
 
     // Check if we have survey response usage data
     const surveyResponsesUsage = billing?.usage_summary?.survey_responses
-    if (!surveyResponsesUsage) {
-        return null
-    }
-
-    const { usage = 0, limit = DEFAULT_SURVEY_RESPONSE_LIMIT } = surveyResponsesUsage
-    const percentageUsed = (usage / limit) * 100
-    const remaining = limit - usage
-
-    // Don't show if no usage and not approaching limit
-    if (usage === 0 && percentageUsed < 80) {
+    if (!surveyResponsesUsage?.usage) {
         return null
     }
 
-    let bannerType: 'info' | 'warning' | 'error' = 'info'
-    let message = `Survey Responses This Month: ${usage} / ${limit}`
-
-    if (percentageUsed >= 100) {
-        bannerType = 'error'
-        message = `You've reached your monthly limit of ${limit} survey responses.`
-    } else if (percentageUsed >= 80) {
-        bannerType = 'warning'
-        message = `You're approaching your monthly limit: ${usage} / ${limit} responses used.`
+    const { usage = 0, limit } = surveyResponsesUsage
+    const percentageUsed = limit ? (usage / limit) * 100 : 0
+
+    let type: 'info' | 'warning' | 'error' = 'info'
+    let message: string
+
+    if (usage === 0) {
+        message = `You have used 0 responses this month.`
+    } else if (limit && usage >= limit) {
+        type = 'error'
+        message = `You have used all ${limit} responses this month. Further responses will not be collected.`
+    } else if (limit && percentageUsed >= 80) {
+        type = 'warning'
+        message = `You have used ${usage} of ${limit} responses this month. You have ${
+            limit - usage
+        } responses remaining.`
+    } else if (limit) {
+        message = `You have used ${usage} of ${limit} responses this month. You have ${
+            limit - usage
+        } responses remaining.`
+    } else {
+        message = `You have used ${usage} responses this month.`
```

Comment:
I think the language here can be improved. Think about the UX people seeing this: `used X responses` doesn't really make a lot of sense for the surveys page.

Instead, we should do something like `you have received X responses this month. Your current limit is Y responses per month.`

we don't really have to use the billing-side of language, since I think we have two goals here:

- People should know how many responses they received
- They should also know how many more they can still receive until they reach their limit

what do you think?